plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/worker/ConsoleReportWorkAction.kt

```diff
@@ -0,0 +1,70 @@
+package org.jlleitschuh.gradle.ktlint.worker
+
+import com.pinterest.ktlint.core.LintError
+import org.gradle.api.GradleException
+import org.gradle.api.file.ConfigurableFileCollection
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.logging.Logging
+import org.gradle.api.provider.Property
+import org.gradle.workers.WorkAction
+import org.gradle.workers.WorkParameters
+import org.jetbrains.kotlin.util.prefixIfNot
+
+@Suppress("UnstableApiUsage")
+internal abstract class ConsoleReportWorkAction : WorkAction<ConsoleReportWorkAction.ConsoleReportParameters> {
+
+    private val logger = Logging.getLogger("ktlint-console-report-worker")
+
+    override fun execute() {
+        val errors = loadErrors(parameters.discoveredErrors.asFile.get())
+
+        if (parameters.outputToConsole.getOrElse(false)) {
+            val verbose = parameters.verbose.get()
+            errors.forEach { lintErrorResult ->
+                val filePath = lintErrorResult.lintedFile.absolutePath
+                lintErrorResult
+                    .lintErrors
+                    .filter { !it.second }
+                    .forEach {
+                        it.first.lintError.logError(filePath, verbose)
+                    }
+            }
+        }
+
+        if (errors.any { it.lintErrors.any { !it.second } } &&
+            !parameters.ignoreFailures.getOrElse(false)
+        ) {
+            val reportsPaths = parameters
+                .generatedReportsPaths
+                .files.joinToString(separator = "\n") { it.absolutePath.prefixIfNot("|- ") }
+
+            throw GradleException(
+                """
+                |KtLint found code style violations. You could find them in following reports:
```

Comment:
```suggestion
                |KtLint found code style violations. Please see the following reports:
```

Reply:
You don't have to change it to exactly this, but 'could' is past-tense. If you want to keep it as you have it, change it to: 'You can find them'. 'You can find' would be present tense.

Reply:
Fixed via b5bdc20df